code/datums/id_trim/jobs.dm

```diff
@@ -722,7 +722,7 @@
 /datum/id_trim/job/prisoner
 	assignment = "Prisoner"
 	trim_state = "trim_warden"
-	orbit_icon = "lock-keyhole"
+	orbit_icon = "lock"
```

Comment:
```suggestion
	orbit_icon = "container-storage"
```
![image](https://user-images.githubusercontent.com/29008542/197047816-933135a4-4650-43b3-a467-1a4d0f93e66d.png)
makes a little more sense imo

Reply:
Agree! Just need to test it, not all icons work even though they should (same issue here!)

Reply:
Won't work, the whole problem is we have the free version of FontAwesome, you need to press the Free filter when looking through them.

Reply:
Good catch haha, thought the issue was a different version of FA